make/modules/java.desktop/lib/Awt2dLibraries.gmk

```diff
@@ -126,8 +126,10 @@ endif
 # https://gcc.gnu.org/bugzilla/show_bug.cgi?id=45977
 ifeq ($(TOOLCHAIN_TYPE), gcc)
   BUILD_LIBAWT_debug_mem.c_CFLAGS := -w
-  # This option improves performance of MaskFill in Java2D by 20% for some gcc
-  LIBAWT_CFLAGS += -fgcse-after-reload
+  # This option forces gcc to generate position independent code, so
+  # that any cleanup of code related to AWT will not change the
+  # generated code and cause performance issues in J2DBench
+  LIBAWT_CFLAGS += -fpic
```

Comment:
We compile with `-fPIC` as a standard flag for position independent code (I just verified that LIBAWT also gets this flag today). My understanding is that `-fpic` may result in smaller/faster code. Overriding with `-fpic` seems like its helping here, but the comment needs to be fixed to explain the actual difference and reason.

Reply:
Thanks @erikj79 for your inputs. I have updated the comment to be relevant to the problem it is resolving.